README.md

```diff
@@ -166,6 +166,21 @@ end
 Calculator.start_link(x, name: :foo)
 ```
 
+This will allow you to pass in the actor process name when instantiated as a worker by the application like so:
```

Comment:
Thank you for the PR!

I would propose to condense this, and add only supervisor application to the block above. So immediately below line 166, you could add something like:

```elixir
# Or in the supervisor specification:
Supervisor.start_link(
  [
    worker(Calculator, [x, [name: :foo]]),
    # ...
  ]
)
```

This seems consistent with the previous code snippet, and I believe it's enough information. What do you think?